scripts/collect.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+from math import log, tan, pi
+from itertools import product
+from argparse import ArgumentParser
+from os.path import join
+
+import tempfile, shutil, urllib.request, io, sys, subprocess
+import unittest, unittest.mock as mock
+
+tile_url = 'https://terrain-preview.mapzen.com/geotiff/{z}/{x}/{y}.tif'
+
+def mercator(lat, lon, zoom):
+    ''' Convert latitude, longitude to z/x/y tile coordinate at given zoom.
```

Comment:
(gdal_merge.py can be gotten as a binary download via Postgres.app, weirdly, so no homebrew or pre-build requirement there)